app/src/main/java/org/fossasia/openevent/fragments/SpeakerFragment.java

```diff
@@ -71,16 +72,27 @@
 
     private int sortType;
 
+    private TextView noSpeakers;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'noSpeakers' could be replaced by a local variable.](https://www.codacy.com/app/triplez/open-event-android/file/2406594559/issues/source?bid=3233850&fileBranchId=3233896#l75)